src/client.ts

```diff
@@ -71,19 +70,21 @@ export type CustomClientOptions = {
  * @property [data] - The data for a POST request.
  * @property [url] - The full url to contact. Will be computed from the portalUrl and endpointPath if not provided.
  * @property [method] - The request method.
+ * @property [headers] - Any request headers to set.
+ * @property [subdomain] - An optional subdomain to add to the URL.
  * @property [query] - Query parameters.
  * @property [extraPath] - An additional path to append to the URL, e.g. a 46-character skylink.
- * @property [headers] - Any request headers to set.
  * @property [responseType] - The response type.
  * @property [transformRequest] - A function that allows manually transforming the request.
  * @property [transformResponse] - A function that allows manually transforming the response.
  */
 export type RequestConfig = CustomClientOptions & {
-  endpointPath: string;
+  endpointPath?: string;
```

Comment:
update the fact that it's optional in the `@property` line in the docstring

Reply:
Good cache.